sormas-app/app/src/main/java/de/symeda/sormas/app/backend/common/DatabaseHelper.java

```diff
@@ -2993,6 +2993,16 @@ public void onUpgrade(SQLiteDatabase db, ConnectionSource connectionSource, int
 				getDao(DiseaseConfiguration.class).executeRaw("ALTER TABLE diseaseConfiguration ADD COLUMN ageGroupsString text;");
 				getDao(DiseaseConfiguration.class).executeRaw("UPDATE diseaseConfiguration SET changeDate = 0;");
 
+			case 337:
+				currentVersion = 337;
+				getDao(DiseaseConfiguration.class)
```

Comment:
This won't work. Column renaming is not supported in the SQLite version we're using. There's two possible workarounds for this; the easier one is to add a new `caseSurveillanceEnabled` column, copy the values from `caseBased` to the new column, and then mark the `caseBased` column as deprecated.

Also, SQLite requires colum names to be written in camel case, otherwise this script won't run through correctly.

Please make sure to test this migration before re-submitting the PR for review :)